cmd/controller/app/controller.go

```diff
@@ -232,7 +234,7 @@ func buildControllerContext(ctx context.Context, stopCh <-chan struct{}, opts *o
 			IssuerAmbientCredentials:        opts.IssuerAmbientCredentials,
 			ClusterResourceNamespace:        opts.ClusterResourceNamespace,
 		},
-		IngressShimOptions: controller.IngressShimOptions{
+		CertificateShimOptions: controller.CertificateShimOptions{
```

Comment:
I remember hearing that "ingress shim" was not an appropriate name and that "certificate shim" made more sense. Could you document that somewhere? E.g. in the website since end-users may also wonder why we are calling it that 😅

Reply:
Yes, after the feedback from the last retro this should not be merged without a matching PR to the documentation